git-keeper.py

```diff
@@ -125,11 +136,12 @@ def main():
         aws_access_key_id, aws_secret_access_key, region_name, endpoint_url
     )
 
-    date = datetime.now().strftime("%Y-%m-%d--%H-%M")
+    logger = logging.getLogger(__name__)
```

Comment:
why are you redefining `logger` here? it is defined in the beginning of the script.